### DISCUSSION TITLE
Error of repeater

### DISCUSSION THREAD

**Discussion by tharryz** [Mygod/VPNHotspot#324] [Q&A]
Hello, the repeater worked perfectly in my mobile Android 10 before, but it suddenly couldn't be connected and showed unable to create P2P (interior error). I tried to clean up misc/wifi and restarted mobile, but it still doesn't work. Could you please help me to solve this problem?
Thanks a lot.

**Comment by Mygod** [comment]
Turn on repeater safe mode, or use a longer SSID.

**Reply by tharryz** [reply]
How to turn on it in safe mode?

**Reply by Mygod** [reply]
Look at settings page in this app.

**Reply by tharryz** [reply]
Sorry, but I can't find safe mode or SSID in it![Screenshot_20220119-150143.png](https://user-images.githubusercontent.com/70935741/150145984-b8f38653-bddf-4f03-a10f-cc44286114ac.png)![Screenshot_20220119-150139.png](https://user-images.githubusercontent.com/70935741/150146009-010a73f4-001c-43ab-a4bf-9fc4799bea85.png)

**Reply by Mygod** [reply]
Scroll down.

**Reply by tharryz** [reply]
There is no safe mode.![Screenshot_20220120-182548.png](https://user-images.githubusercontent.com/70935741/150390614-d130d7a3-a472-4ff4-a38b-5f6de3a8770c.png)

**Reply by tharryz** [reply]
![Screenshot_20220120-182920.png](https://user-images.githubusercontent.com/70935741/150390786-3734b136-14a2-4794-b55d-f21cbd7349bd.png)

**Reply by Mygod** [reply]
What is your Android version and Android security update?

**Reply by Mygod** [reply]
SSID changes can be made via clicking the button top right in the tethering tab.

**Reply by tharryz** [reply]
> What is your Android version and Android security update?

Android 10 and security update is Feb 5th. 2020

**Comment by Mygod** [comment]
In that case the cause might be something else. Did you update the system before repeater stops working? Please try to turn on repeater and then export debug information?

**Reply by tharryz** [reply]
- - No, I didn't update the system.  ![Screenshot_20220121-220633.png](https://user-images.githubusercontent.com/70935741/150600183-c3106f08-c4cd-46db-adcd-5c8ae3f92834.png)

**Reply by tharryz** [reply]
I found that I could open the repeater as long as I cleared the app storage data, but it can be only turned on once, and then I need to clear the data again. I think it's a problem with P2P historical address records. Could you please add some button to clean it？

**Reply by Mygod** [reply]
The logs you posted was irrelevant. Could you please post the full logs if possible?

**Reply by tharryz** [reply]
![Screenshot_20220124-183528.png](https://user-images.githubusercontent.com/70935741/150834852-1be7611c-4741-4448-9e47-57229fc867ec.png)![Screenshot_20220124-183503.png](https://user-images.githubusercontent.com/70935741/150834866-c37e1b05-f918-431b-874e-d5088e6b3134.png)
How about these?

**Reply by Mygod** [reply]
Post full.
